charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.jupyterNotebook.enabled -}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: jupyter-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.jupyterNotebook.serviceAccount.name }}
```

Comment:
What if `.Values.jupyterNotebook.serviceAccount.name` is empty? The user may ask to `create` but not provide a name (by mistake or not). Should we use a default name here in that case?